### PR TITLE
Revise pack overview doc

### PR DESCRIPTION
**Pull request by ritawatson**
This PR rewrites the Pack Integrations doc and changes the nav pane label from "Pack Integrations" to "Packs List". Alternatively, it can just be "Packs" but labeling it as "Packs List" distinguishes it from "Registries and Packs".

The 06-integrations.mdx file contains  <Content> and </Content> elements. Are these necessary and can they be deleted? Or are they related to the search bar?

Note: This PR doesn't present the Integrations filter button (that's in the spectro K8s dashboard PR).

[Doc Preview](https://deploy-preview-1112--docs-spectrocloud.netlify.app/integrations)
See also the "Stand-Alone and Integrated Packs" doc in the nav pane below Packs List.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1112#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations.mdx
Comment (karl-cardenas-coding) [inline content/docs/06-integrations.mdx:23]:
```suggestion
When you choose **Infrastructure** or **Add-on**, Palette presents only packs that provide functionality for the selected pack type. When you choose **Full**, Palette presents all the packs so you can build your cluster profile from the base layer up. To learn more about cluster profiles, check out the [Cluster Profiles](/cluster-profiles) guide.     
```
Removed extra whitespace

File: content/docs/06-integrations/01-standalone-integrated-packs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/01-standalone-integrated-packs.md:25]:
```suggestion
<WarningBox>

We recommend using the Pack defaults. Default settings provide best practices for your clusters. Changing the default settings can introduce misconfigurations. Carefully review the changes you make to a pack. 

</WarningBox>
```
Not every pack is configured with best security practices, so I changed the language to make it more general.

File: content/docs/06-integrations/01-standalone-integrated-packs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/01-standalone-integrated-packs.md:3]:
```suggestion
metaTitle: "Stand-alone and Integrated Packs"
```

File: content/docs/06-integrations/01-standalone-integrated-packs.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/01-standalone-integrated-packs.md:19]:
```suggestion
Say you want to add the Kubernetes dashboard to your cluster profile. Typically, this would require manually adding a dependency proxy pack to enable the use of a reverse proxy with a Kubernetes cluster. It also requires configuring third-party authentication through OIDC.
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good, just some minor suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1112#pullrequestreview-1287518331" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1112#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-2/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 5effe04d9901f583ed7935ea1efc114aefcfe573 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1112/librarium_build/1623015145217200128) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
